common/adminBlock/EntryRemoveFederatedServer.go

```diff
@@ -10,23 +10,25 @@ import (
 	"github.com/FactomProject/factomd/common/primitives"
 )
 
-// DB Signature Entry -------------------------
+// RemoveFederatedServer is an admin block entry which instructs factomd to remove a federated server at an upcoming block height
```

Comment:
This will also remove an audit server. When we first made the remove, we had 2, but then we realized, if you are removing an ANO, you want him out regardless of his position.

An AddAuditServer is a demotion. A remove is a complete remove from the authority set.

Reply:
Great, added these comments